### ISSUE TITLE
UnknownServiceException: No service of type StyledTextOutputFactory available with Gradle 3.0

### ISSUE THREAD

**Issue by szpak** [allegro/axion-release-plugin#150]
Axion 1.3.6 is not compatible with Gradle 3.0:
> Caused by: org.gradle.internal.service.UnknownServiceException: No service of type StyledTextOutputFactory available in ProjectScopeServices.
on `currentVersion` task execution.

Reading the [forum](https://discuss.gradle.org/t/bug-in-gradle-2-14-rc1-no-service-of-type-styledtextoutputfactory/17638/4?u=marcin) the Gradle team considers `StyledTextOutputFactory` as their internal API and does not provide anything as a replacement. Probably it would be necessary to remove its usage and at most comment in the forum thread that there should be a replacement (or expect this class in two different packages for compatibility with Gradle 2.x and 3.x).

Longer stacktrace:
```
:currentVersion FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':currentVersion'.
> No service of type StyledTextOutputFactory available in ProjectScopeServices.

* Try:
Run with --info or --debug option to get more log output.

* Exception is:
org.gradle.api.tasks.TaskExecutionException: Execution failed for task ':currentVersion'.
(...)
Caused by: org.gradle.internal.service.UnknownServiceException: No service of type StyledTextOutputFactory available in ProjectScopeServices.
	at org.gradle.internal.service.DefaultServiceRegistry.getServiceProvider(DefaultServiceRegistry.java:436)
	at org.gradle.internal.service.DefaultServiceRegistry.doGet(DefaultServiceRegistry.java:426)
	at org.gradle.internal.service.DefaultServiceRegistry.get(DefaultServiceRegistry.java:410)
	at org.gradle.internal.service.ServiceRegistry$get.call(Unknown Source)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:48)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:113)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:125)
	at pl.allegro.tech.build.axion.release.OutputCurrentVersionTask.output(OutputCurrentVersionTask.groovy:21)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.gradle.internal.reflect.JavaMethod.invoke(JavaMethod.java:75)
	at org.gradle.api.internal.project.taskfactory.DefaultTaskClassInfoStore$StandardTaskAction.doExecute(DefaultTaskClassInfoStore.java:133)
	at org.gradle.api.internal.project.taskfactory.DefaultTaskClassInfoStore$StandardTaskAction.execute(DefaultTaskClassInfoStore.java:126)
	at org.gradle.api.internal.project.taskfactory.DefaultTaskClassInfoStore$StandardTaskAction.execute(DefaultTaskClassInfoStore.java:115)
	at org.gradle.api.internal.AbstractTask$TaskActionWrapper.execute(AbstractTask.java:623)
	at org.gradle.api.internal.AbstractTask$TaskActionWrapper.execute(AbstractTask.java:606)
	at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeAction(ExecuteActionsTaskExecuter.java:80)
	at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeActions(ExecuteActionsTaskExecuter.java:61)
	... 71 more
```

**Comment by adamdubiel** [comment]
Wow, i didn't know it was released :) I will take a look at it today, this styled output is not a must.

**Comment by adamdubiel** [comment]
Uh, this change is awful - Gradle should provide some API for nice text output, being a primarily console-based tool.. anyway: bye bye nice output, now it will be simple black&white one. I implemented changes, tests are green, but need some empirical testing, tbc in the evening.

**Comment by adamdubiel** [comment]
I released the 1.4.0 version compatible with Gradle 3.0.
